### ISSUE TITLE
Add label for filename input field

### ISSUE THREAD

**Issue by IzaCoder** [IzaCoder/File-Structure-Organizer#7]
As of now, there is no label for the filename input field, which decreases accessibility (which is a top priority)

**Comment by IzaCoder** [comment]
Working on it!

**Comment by IzaCoder** [comment]
The label is added and it works!
